SwiftTweaks/TweakTableCell.swift

```diff
@@ -242,6 +275,15 @@ internal final class TweakTableCell: UITableViewCell {
 
 
 	// MARK: Events
+	@objc private func segmentedControlChanged(_ sender: UISegmentedControl) {
+		switch viewData! {
+		case let .stringList(_, defaultValue: defaultValue, options: options):
+			viewData = .stringList(value: options[sender.selectedSegmentIndex], defaultValue: defaultValue, options: options)
+			delegate?.tweakCellDidChangeCurrentValue(self)
+		default:
+			assertionFailure("Shouldn't be able to update segmented control if view data isn't StringList type")
```

Comment:
I was just matching the rest of this file, which uses `assertionFailure`s instead of `fatalError`s